projects/angular/src/forms/datepicker/date-input.ts

```diff
@@ -192,6 +194,15 @@ export class ClrDateInput extends WrappedFormControl<ClrDateContainer> implement
     }
   }
 
+  private triggerControlValidation() {
+    if (this.datepickerHasFormControl()) {
+      this.control?.control?.updateValueAndValidity({ emitEvent: false }); // Added emitEvent purposefully inorder to prevent unnecessary valuechange triggers
```

Comment:
```suggestion
      // Set `emitEvent` to false to prevent unnecessary value change event. Status change event will be emitted below by `setErrors`.
      this.control.control?.updateValueAndValidity({ emitEvent: false });
```

The code is already checking if a control exists. 

Also, I'd put the comment on the preceding line for readability (instead of one long line) and clarify about the status change event here as well.